AudioReactiveShapes/.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
You need a .gitignore file so you aren't committing your local vscode config files